pineappl_py/.gitignore

```diff
@@ -21,8 +21,13 @@ coverage.xml
 .coverage
 
 # Sphinx documentation
-docs/_build/
-doc/_build/
+# Ignore auto generated module references
+# TODO: while apidoc does not work, we need to preserve that folder
+# docs/source/modules
+# ignore temporary build files
+docs/build/
+# Ignore generated sphinx-bibtex file
+docs/source/bibtex.json
```

Comment:
We don't use bibtex (and I don't think we ever did), so let's remove this line.

Reply:
done in 77995e07 (as you can guess from earlier this is just copied from eko)